src/BlueberryUpgrades.ts

```diff
@@ -658,12 +680,16 @@ export const saveBlueberryTree = async (input: number, previous: BlueberryOpt) =
   player.blueberryLoadouts[input] = getBlueberryTree()
   // eslint-disable-next-line
   createLoadoutDescription(input, player.blueberryLoadouts[input])
+
+  updateLoadoutHoverClasses()
 }
 
 export const createLoadoutDescription = (input: number, modules: BlueberryOpt) => {
 
   let str = ''
   for (const [key, val] of Object.entries(modules)) {
+    if (!val) continue
```

Comment:
when is this possible? Leaving a comment would be nice

Reply:
Comment added.    tl,dr:  Point 3 in the description.